crates/stackable-operator/src/builder/pod/mod.rs

```diff
@@ -673,6 +720,7 @@ mod tests {
                     .with_config_map("configmap")
                     .build(),
             )
+            .unwrap()
```

Comment:
We should handle the error, or use `expect("...")`

Reply:
Pls heave a look at 53ef278de66ca07c9680810230235ee7a8b260a9

Reply:
It's in tests